Model/Library.cs

```diff
@@ -95,6 +94,28 @@ public static void UpdateCache()
         }
       }
 
+      // Search for JCF directories.
+      var dirs = Directory.GetDirectories(Properties.Settings.Default.TrackPath, "*.jcf");
```

Comment:
This is fine for OSX users, but on Windows the default behaviour of jammit is to remove the ".jcf" so this should also be searching for folders that are just GUID strings